src/formatter.py

```diff
@@ -34,7 +37,7 @@ def format(self):
                 logger.info(f' Downloaded file path: {downloaded_file_path}')
                 f = Formatter(workdir=DOWNLOAD_FILE_PATH, file_path=downloaded_file_path, prefix=self.prefix)
                 formatter = f.osw2osm()
-                OSWFormat.clean_up(downloaded_file_path)
+                # OSWFormat.clean_up(downloaded_file_path)
```

Comment:
why is clean up removed?

Reply:
now self.format is in Thread (async in nature)and we have same file name for all the events - 
Thread 1 - calls the `self.format` 
Thread 2 - calls the `self.format` 

What happens- when Thread 2 is processing the file, Thread 1 deletes that